src/components/richText/DefaultStyling.ts

```diff
@@ -5,14 +5,16 @@ import { EditorThemeClasses } from "lexical";
  * Token Types come from the PrismJS library, which Lexical uses to power Code Blocks.
  * The Types include Comment, Variable, Operator, etc.
  */
-const attributeTokenStyles = "text-sky-700";
-const propertyTokenStyles = "text-pink-800";
-const commentTokenStyles = "text-slate-500";
-const functionTokenStyles = "text-rose-500";
-const selectorTokenStyles = "text-lime-600";
-const variableTokenStyles = "text-amber-500";
-const operatorTokenStyles = "text-yellow-700";
-const punctuationTokenStyles = "text-neutral-400";
+const stylesToTokenTypes = {
```

Comment:
I was going to say we can add a`Record<string, EditorThemeClassName` type here that makes sure we also specify correct token type, but `EditorThemeClassName` is just a `string` and not actually useful...